Projects/TEAM-1/Turbal Y/Task6_8/Задача6_8/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+static class zadachi
```

Comment:
It's ok for our purposes, but the best approach is to store each class in a separate file.